src/pathogen_cluster/pathogen_cluster.py

```diff
@@ -0,0 +1,47 @@
+import hdbscan
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+import numpy as np
+
+def pathogen_cluster(args):
+
+    embedding_df = pd.read_csv(args.embedding, index_col=0)
+
+    clustering_parameters = {
+        **({"min_cluster_size": args.min_size} if args.min_size is not None else {}),
+        **({"min_samples": args.min_samples} if args.min_samples is not None else {}),
+        **({"cluster_selection_epsilon": args.distance_threshold} if args.distance_threshold is not None else {})
+    }
+
+    clusterer = hdbscan.HDBSCAN(**clustering_parameters)
+    clusterer_default = hdbscan.HDBSCAN()

```

Comment:
I think this "default" clusterer and its outputs were useful for our initial exploration of HDBSCAN, but now that we will have a separate cluster command that makes it easy to cluster with different settings, we can safely drop this default clusterer from this code.